README.rdoc

```diff
@@ -19,22 +19,36 @@ Ver. 1.0.x is not 100% backward compatible with 0.3.x. Some methods have been re
 
 First, follow "Create a client ID and client secret" in {this page}[https://developers.google.com/drive/web/auth/web-server] to get a client ID and client secret for OAuth. Set "Application type" to "Other" in the form to create a client ID if you use GoogleDrive.saved_session method as in the example below.
 
+
+The client ID and client secret you obtained in the step above.
```

Comment:
Connection between this sentence and the next sentence is a bit unnatural. Can you simply say e.g.,

> Create config.json file which contains the generated client ID and client secret, which looks like this: